pyanaconda/ui/gui/spokes/lib/lang_locale_handler.py

```diff
@@ -40,7 +40,7 @@ class LangLocaleHandler(object):
 
     """
 
-    def __init__(self, payload):
+    def __init__(self, data, storage, payload, instclass):
```

Comment:
Shouldn't it be better to just pass intclass and payload here?

Reply:
Fixed.